packages/picker/src/Picker.ts

```diff
@@ -406,6 +411,16 @@ export class PickerBase extends SizedMixin(Focusable) {
                           ></sp-icon-alert>
                       `
                     : nothing}
+                ${this.loading
+                    ? html`
+                          <sp-progress-circle
+                              label="Loading picker"
```

Comment:
How do you localize stuff in this project? I to search some examples, but found some unlocalized strings 😅 
<img width="1218" alt="Screenshot 2023-04-06 at 18 16 19" src="https://user-images.githubusercontent.com/40989335/230423717-3da4971a-8b65-4add-9f09-dc6e47dc7db9.png">

Reply:
Yes, I see that `File` is the default, but if the user doesn't set another value and relies on the default, the default is not localized 😅

If I remove the aria-label, the a11y test fails, with the following message:
```
Error: Accessibility Violations
      ---
      Rule: aria-progressbar-name
      Impact: serious
      ARIA progressbar nodes must have an accessible name (https://dequeuniversity.com/rules/axe/4.4/aria-progressbar-name?application=axeAPI)
      
      Issue target: #picker,sp-progress-circle
      Context: <sp-progress-circle indeterminate="" dir="ltr" size="m" role="progressbar"></sp-progress-circle>
      Fix any of the following:
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
      ---
```

So if we do not want to localize something, the only solution would we to use `aria-labelledby` and tie it with the button's content and hope that the user updates the button with a suggestive message when loading. WDYT?